### DISCUSSION TITLE
Can I download it in 1080p?

### DISCUSSION THREAD

**Discussion by MinhHieu012** [heliomarpm/udemy-downloader-gui#187] [General]
How can I download it in 1080p?

**Comment by heliomarpm** [comment]
> How can I download it in 1080p?

Set highest in settings

**Reply by MinhHieu012** [reply]
not working! it only download it in 720p

**Comment by FAUSTOCG** [comment]
Hello my friend first I want to thank you for this great app udeler, but wanted to let you know that there is a glitch i have downloaded a full course that just has been released but when playing the videos the audios have glitches but when playing them directly from the web audio has no glitches I download it again in different computers with the same audio glitches bough another course and it happened again videos play great buy audios have glitches or could be some sort of DRM implementation by Udmy?

thanks for keeping this great alive app have a great week

update: I just checked again and the problem is when downloading 1080 resolution in 720 audio has no issue, but when downloading
at 1080 down speed drops too much and audio from videos have glitches

**Reply by heliomarpm** [reply]
Hello my friend, how are you? Thank you for the diagnosis, it helps me a lot to try to find the problem.

**Comment by FAUSTOCG** [comment]
No, thank you! for keep this great app working with out your efforts many people would not be able to have their paid courses offline in their computers for when no internet access is availabe like me.

Good luck hope you will find the fix soon.

Have a great weekend!


________________________________
